plugins/hydra_ax_sweeper/tests/test_ax_sweeper_plugin.py

```diff
@@ -107,6 +107,33 @@ def test_jobs_configured_via_config(hydra_sweep_runner: TSweepRunner) -> None:
         assert math.isclose(best_parameters["quadratic.y"], -1.0, abs_tol=1e-4)
 
 
+def test_jobs_configured_via_config_logscale(hydra_sweep_runner: TSweepRunner) -> None:
+    sweep = hydra_sweep_runner(
+        calling_file="tests/test_ax_sweeper_plugin.py",
+        calling_module=None,
+        task_function=quadratic,
+        config_path="config",
+        config_name="config.yaml",
+        overrides=[
+            "hydra/launcher=basic",
+            "hydra.sweeper.ax_config.max_trials=100",
+            "hydra.sweeper.ax_config.early_stop.max_epochs_without_improvement=3",
+            "params=logscale",
+        ],
+    )
+    with sweep:
+        assert sweep.returns is None
+        returns = OmegaConf.load(f"{sweep.temp_dir}/optimization_results.yaml")
+        assert isinstance(returns, DictConfig)
+        assert returns["optimizer"] == "ax"
+        assert len(returns) == 2
+        best_parameters = returns.ax
+        # the point of this test is to make sure that Ax runs with logscale.
+        # It's hard to get the correct answer just trying values.
+        assert "quadratic.x" in best_parameters
+        assert "quadratic.y" in best_parameters
```

Comment:
i set 
```
quadratic.x:
  type: range
  bounds: [1e-6, 1]
  log_scale: True
```
for log scale, and `math.isclose(best_parameters["quadratic.x"], 0.0, abs_tol=1e-4)` returns True :)

Reply:
Sure, but see the above comments. I don't think it's right to actually expect the correct answer to this optimization problem with the sweeper in this way.